src/labelling.jsx

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+import React from 'react';
+import classnames from 'classnames';
+
+var getSlidesCount = function (spec) {
+  var slides;
+  slides = Math.ceil(spec.slideCount / spec.slidesToScroll);
+  return slides;
+};
+
+
+export var Labelling = React.createClass({
+
+  render: function () {
+    var template, className, label;
+
+    if (this.props.labellingTemplate) {
```

Comment:
Use ES6 destructuring. `const { labellingTemplate, slideCount, ... } = this.props;` 

Just use `const` all over the place. I'll do for the rest of the library later.

May be use labelTemplateFn / labelTemplateFunc instead of labellingTemplate. Instead of Labelling just use Label.